create.go

```diff
@@ -52,7 +57,124 @@ var createCommand = cli.Command{
 		},
 	},
 	Action: func(context *cli.Context) error {
-		// TODO
-		return nil
+		return create(context.String("container-id"),
+			context.String("bundle"),
+			context.String("console"),
+			context.String("pid-file"))
 	},
 }
+
+func create(containerID, bundlePath, console, pidFilePath string) error {
+	// Checks the MUST and MUST NOT from OCI runtime specification
+	if err := checkCreateParamsIntegrity(containerID, bundlePath); err != nil {
+		return err
+	}
+
+	podConfig, err := getPodConfig(bundlePath, containerID, console)
+	if err != nil {
+		return err
+	}
+
+	pod, err := vc.CreatePod(podConfig)
+	if err != nil {
+		return err
+	}
+
+	shimInfo, err := getShimInfo(pod)
+	if err != nil {
+		return err
+	}
+
+	// Start the shim to retrieve its PID.
+	pid, err := startShim(shimInfo)
+	if err != nil {
+		return err
+	}
+
+	if err := storeShimPid(pod, pid); err != nil {
+		return err
+	}
+
+	// Creation of PID file has to be the last thing done in the create
+	// because containerd considers the create complete after this file
+	// is created.
+	if err := createPIDFile(pidFilePath, pid); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func checkCreateParamsIntegrity(containerID, bundlePath string) error {
```

Comment:
Could we have an `oci.go` file where we manage the spec compliance parts ?
So all the parameters validation (And not integrity ;)) routines should be added there.

Also please not that except for `create`, all other commands in this PR only validate the container ID. So having a `validContainer(containerID, command string) (bool, error)` generic routine would make the code overall more readable.